### DISCUSSION TITLE
1.7.2

### DISCUSSION THREAD

**Discussion by aviraxp** [LSPosed/LSPosed#1648] [New Release]
### Changelog
- Fix black theme when using customized color
- Show `pm hide`-ed modules in the module list
- Suppress notification of `pm unhide`-ing modules
- Fix updating in-app may update to an older version
- Fix daemon crash if any overlay Magisk module is used
- Dynamic color shortcut for Android 12+
- Machine translation for incomplete translation

### 更新日志
- 修复自定义主题色下的纯黑主题
- 模块列表显示被 `pm hide` 的模块
- 不显示 `pm unhide` 模块时的通知
- 修复应用内更新可能更新到旧版本的问题
- 修复使用了 overlay 的 Magisk 模块时候守护进程的崩溃
- Android 12+ 的快捷方式使用动态颜色
- 未完成翻译使用机器翻译补全

<hr /><em>This discussion was created from the release <a href='https://github.com/LSPosed/LSPosed/releases/tag/v1.7.2'>1.7.2</a>.</em>

**Comment by apkpenetrator** [comment]
it's not working in my device

**Comment by apkpenetrator** [comment]
it's activated but doesn't work

**Comment by Zaharmanda** [comment]
Yhjj

**Comment by ASL-Vanity** [comment]
什么时候可以适配13啊？

**Reply by yujincheng08** [reply]
https://github.com/LSPosed/LSPosed/commit/b0d1e7a2bb357f539f0d41d0dfb55f483cdaec8a
https://github.com/LSPosed/LSPosed/commit/dcd29b8751e92b2f36569fb844092afab3268cb6
https://github.com/LSPosed/LSPosed/commit/2b70e64e6ddfc9cbafbe403f37baacb332f136da
https://github.com/LSPosed/LSPosed/commit/ef1439a3f36dc57ad7b72161a759e5960b1ce91c

**Reply by yujincheng08** [reply]
有没有一种可能，发布当天就适配了

**Reply by ASL-Vanity** [reply]
没呀，我就是13,刷了没反应

**Reply by yujincheng08** [reply]
![image](https://user-images.githubusercontent.com/5022927/155709062-c7e8dffd-4f7f-46bc-b920-69120660e1b6.png)

**Reply by ASL-Vanity** [reply]
![Screenshot_20220225-195003.png](https://user-images.githubusercontent.com/53140010/155710736-73209135-9181-4172-aadc-516323b3394a.png)

**Reply by yujincheng08** [reply]
用 CI 啊😅

**Comment by sionnx** [comment]
你好我在写module的时候，发现这个版本的zygisk模式下  XposedBridge.log 打不出来日志，我拿之前写的xposed模块试了一下，也打不出来 ，师傅们可以看看 我之前写的这个 https://github.com/runtimem/Anti-Screenshot/  ，有源代码和之前编译的apk，观察一下行为

**Reply by sionnx** [reply]
实测，可能是zygisk不稳定造成的，重新刷了一下环境可以了

**Comment by paynexss** [comment]
pixel 3xl android12 magisk24.3，lsposed1.7.2-Zygisk，为什么每次安装模块都需要重启才能生效额
